driver/amdair_process.c

```diff
@@ -103,12 +103,21 @@ int amdair_process_create(const struct task_struct *process,
 
 int amdair_process_release_resources(struct amdair_process *air_process)
 {
-	int i = 0;
+	int i = 0, q = 0;
+	uint32_t queue_id = 0;
 
 	for (i = 0; i < air_process->num_proc_devs; ++i) {
 		amdair_doorbell_release(air_process->proc_devs[i].dev,
 					air_process->proc_devs[i].db_page_id);
 		idr_destroy(&air_process->proc_devs[i].alloc_idr);
+
+		// Releasing all queues used by the process
```

Comment:
Similar to the assignment, can you add this to:

amdair_process_device_release_queues(&air_pd) and have it iterate and free the queues from its list.

Add space between if (